Dockerfile_k8s_gpu

```diff
@@ -0,0 +1,58 @@
+# TODO(romilb) - The base image used here (ray) is very large (11.4GB).
+#  as a result, this built image is about 13.5GB. We need to pick a lighter base
+#  image.
+FROM rayproject/ray:2.4.0-gpu
```

Comment:
nit: should we explicitly use `ray:2.4.0-py310-gpu` as we are using py310 for other clouds. : )

Reply:
Done! I've updated both the CPU and GPU images.